guides/common/modules/proc_registering-hosts-on-aws-to-foreman.adoc

```diff
@@ -0,0 +1,8 @@
+[id="registering-hosts-on-aws-to-{project-context}"]
+= Registering hosts on AWS to {Project}
```

Comment:
I think it would be more user-friendly to include this as a step in the "Installing Foreman server on AWS" and "Installing Smart Proxy server on AWS" procedures. Users are unlikely to want to install a server/proxy server without continuing to register the hosts so in my mind, it's all one procedure.